### PR TITLE
[swweg] adding new character sheet stat block files

### PR DESCRIPTION
**Pull request by CBairdAsh**
## ADD :
    - Schema\swweg\character-swweg.yml : adding a new data model for character data
    - css\swweg\swweg.css: adding a css stylesheet to support stat block display
    - Display\swweg\character.html.twig: added new character sheet
    - Display\swweg\swweg-base.html.twig: added new base to support character sheet
    - Display\swweg\swweg-macros.html.twig: added macros to support character sheet rendering
## MOD :
    - Display/dch/character.html.twig : replaced span tag with basic parenthesis for better appearance
    - Display/dch/dch-macros.html.twig : Corrected misspelling of the abbreviation 'approx'
    - Display/dch/dch-macros.html.twig : Fixed bug in macro annwealth so that it wouldn't accidentally show the higher tier text along with the correct wealth text.
    - Display/dch/dch-macros.html.twig : Fixed bug in attribute breakdown macro, stat ranges were not working
    - css/dch/dch.css : added style adjustments to improve look on final character sheet use.

### CODE REVIEW

File: Display/swweg/swweg-base.html.twig [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    		<div class="col-md-12">
+    			<div class="row">
+    				<div class="col-md-6 char_name">
+              {{ variables.name }}
+    				</div>
+    				<div class="col-md-6">
+              <div class="row">
+                <div class="col-md-12">
+        					<div class="flex-base flex-row">
+                    {{ macros.quik_stat(variables.cpts,'Character Points') }}
```
Comment (GorkamWorka) [inline Display/swweg/swweg-base.html.twig]: quik_stat ? isn't more like quick_stat ?

Comment (CBairdAsh) [reply Display/swweg/swweg-base.html.twig]: Yep. I was just typing too fast. But I misspelled the macro name as 'quik_stat' too! I'll fix the spelling

**Comment by CBairdAsh** [comment]
The DCH change wasn't mentioned as I was doing too much at once. I'll adjust.

File: Display/swweg/swweg-base.html.twig (hunk 1 above)
Comment (CBairdAsh) [reply Display/swweg/swweg-base.html.twig]: Spelling corrected

Comment (CBairdAsh) [reply Display/swweg/swweg-base.html.twig]: Correction should be checked in.

**Pull request closed without merging**